javascript/express/security/injection/tainted-sql-string.yaml

```diff
@@ -25,11 +25,11 @@ rules:
     - vuln
     likelihood: HIGH
     impact: MEDIUM
-    confidence: MEDIUM
+    confidence: LOW
   languages:
   - javascript
   - typescript
-  severity: ERROR
+  severity: WARNING
```

Comment:
Why reducing severity? High likelihood x Medium impact should give Error.
(the severity is independent from the confidence)

Reply:
The rules generates lots of noise, the goal is to reduce the severity.